Bottom: d9767396d98f20c4eb5cef8235a6e0308098a77c
Top:    db926208febc92c0d5dbec1391c6f358b9a6174c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-30 00:26:20 +0100

Refresh of hid-ps-don-t-use-the-core-s-dr

---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index caf792a..3ca86b9 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -1242,7 +1242,7 @@ ps_fill_polygon (hidGC gc, int n_coords, Coord *x, Coord *y)
 }
 
 static void
-draw_poly (hidGC gc, PolygonType * poly, const BoxType * clip_box)
+ps_fill_pcb_polygon (hidGC gc, PolygonType * poly, const BoxType * clip_box)
 {
   /* Ignore clip_box, just draw everything */
 
@@ -1506,6 +1506,7 @@ void ps_ps_init (HID *hid)
   hid->draw_rect          = ps_draw_rect;
   hid->fill_circle        = ps_fill_circle;
   hid->fill_polygon       = ps_fill_polygon;
+  hid->fill_pcb_polygon   = ps_fill_pcb_polygon;
   hid->fill_rect          = ps_fill_rect;
   hid->calibrate          = ps_calibrate;
   hid->set_crosshair      = ps_set_crosshair;
